setting up Miniconda...
activating virtual env...
current conda environment: basic310
activating ssh connection...
Agent pid 2413207
Identity added: /home/11864265/.ssh/id_ed25519 (11864265@fnwi-h0.science.uva.nl)
parsing input arguments...
Branch: feature/retrain_grammar
fetching and pulling latest changes from remote dir...
Already up to date.
running perturbed masking...
Traceback (most recent call last):
  File "/home/11864265/Documents/thesis_code/reduce_grammar/generate_subset.py", line 1, in <module>
srun: error: fnwi-cn005: task 0: Exited with exit code 1
    import nltk
ModuleNotFoundError: No module named 'nltk'
adding, committing, and pushing results to remote dir...
fatal: Invalid path '/home/11864265/thesis_code': No such file or directory
On branch feature/retrain
Your branch is up to date with 'origin/feature/retrain'.

Untracked files:
  (use "git add <file>..." to include in what will be committed)
	jobs/slurm_output_3233.out
	jobs/slurm_output_3234.out
	jobs/slurm_output_3235.out
	jobs/slurm_output_3236.out

nothing added to commit but untracked files present (use "git add" to track)
Everything up-to-date
done.
